documentation/modules/con-kafka-logging.adoc

```diff
@@ -0,0 +1,90 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+[id='con-kafka-logging-{context}']
+= Kafka loggers
+
+Kafka has its own configurable loggers:
+
+* `kafka.root.logger.level`
+* `log4j.logger.org.I0Itec.zkclient.ZkClient`
+* `log4j.logger.org.apache.zookeeper`
+* `log4j.logger.kafka`
+* `log4j.logger.org.apache.kafka`
+* `log4j.logger.kafka.request.logger`
+* `log4j.logger.kafka.network.Processor`
+* `log4j.logger.kafka.server.KafkaApis`
+* `log4j.logger.kafka.network.RequestChannel$`
+* `log4j.logger.kafka.controller`
+* `log4j.logger.kafka.log.LogCleaner`
+* `log4j.logger.state.change.logger`
+* `log4j.logger.kafka.authorizer.logger`
+* `zookeeper.root.logger`
```

Comment:
Is this an actual logger for KAfka? Or is this only for Zookeeper? I guess the other. I think we need to split this into two separate lists - for KAfka and for Zoo.

Reply:
I split it off from the list.